### PR TITLE
Fix camera uniform type

### PR DESCRIPTION
**Pull request by brieucdesoutter**
Hi, 
First thanks for your work, I love it and it makes wgpu really approchable. While following the beginner tutorial about "Uniform buffer and 3d camera" I run into this error: 

```
[2021-09-06T11:27:29Z ERROR wgpu_core::device] Failed to parse WGSL code for Some("Shader"): unknown type: 'Camera'
[2021-09-06T11:27:29Z ERROR wgpu::backend::direct] wgpu error: Validation Error

    Caused by:
        In Device::create_shader_module
          note: label = `Shader`
        Failed to parse WGSL
```

I could fix it by modifying the first few lines of the shader from 

```wgsl
[[block]]
struct CameraUniform {
    view_proj: mat4x4<f32>;
};
[[group(1), binding(0)]]
var<uniform> camera: Camera;
```

to 

```wgsl
[[block]]
struct CameraUniform {
    view_proj: mat4x4<f32>;
};
[[group(1), binding(0)]]
var<uniform> camera: CameraUniform;
```

### CODE REVIEW

**Comment by brieucdesoutter** [comment]
It turns out the shader code shown in the doc uses `CameraUniform` for the struct name but `Camera` for the type of the `camera` uniform whereas the code use `Camera` for the struct name and var type (no error). I reverted my previous change to use the name used in code in the README.md (if that make sense...)

File: docs/beginner/tutorial6-uniforms/README.md [hunk 1]
```diff
@@ -230,11 +230,11 @@ Modify the vertex shader to include the following.
 // Vertex shader
 
 [[block]] // 1.
-struct CameraUniform {
+struct Camera {
     view_proj: mat4x4<f32>;
 };
 [[group(1), binding(0)]] // 2.
-var<uniform> camera: CameraUniform;
+var<uniform> camera: Camera;
```
Comment (sotrh) [inline docs/beginner/tutorial6-uniforms/README.md]: I want the name of the struct in WGSL to match the struct in Rust. While not strictly necessary, this makes it a little more clear how they match up.

**Review by sotrh — CHANGES_REQUESTED** [review]

File: docs/beginner/tutorial6-uniforms/README.md (hunk 1 above)
Comment (VillSnow) [reply docs/beginner/tutorial6-uniforms/README.md]: #226 

> @sotrh
At the moment docs/beginner/tutorial6-uniforms/README.md does not match with code/beginner/tutorial6-uniforms/src/shader.wgsl already.
>
> code/ does not raises error while docs/ does. So I didn't change code/ and match docs/ to code/.
>
> Do you mean both definition and usage of Camera in code/**/shader.wgs is incorrect and must be CameraUniform?

Comment (sotrh) [reply docs/beginner/tutorial6-uniforms/README.md]: Yeah the code is incorrect.

Comment (sotrh) [reply docs/beginner/tutorial6-uniforms/README.md]: Basically it should be `CameraUniform` all around.

**Comment by sotrh** [comment]
It seems like this has been fixed in #228

**Pull request closed without merging**